### PR TITLE
(docs) Add a sentence about smoothing to the scalars tutorial.

### PR DESCRIPTION
**Pull request by bileschi**
Docs cleanup:  Adds a note about how the smoothing parameter may affect the appearance of the learning rate graph.

### CODE REVIEW

File: docs/scalars_and_keras.ipynb [hunk 1]
```diff
@@ -424,7 +424,9 @@
       "source": [
         "Using the \"Runs\" selector on the left, notice that you have a ```<timestamp>/metrics``` run. Selecting this run displays a \"learning rate\" graph that allows you to verify the progression of the learning rate during this run. \n",
         "\n",
-        "You can also compare this run's training and validation loss curves against your earlier runs."
+        "You can also compare this run's training and validation loss curves against your earlier runs.",
+        "\n",
+        "You might also notice that the learning rate schedule returned descrete values, depending on epoch, but the learning rate plot may appear smooth.  TensorBoard has a smoothing parameter that you may need to turn down to zero to see the unsmoothed values. \n"
```
Comment (nfelt) [inline docs/scalars_and_keras.ipynb:429]: typo: discrete

**Review by nfelt — APPROVED** [review]
